src/content/docs/get-started/storefront-compatibility.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: Storefront Compatibility Package
+description: Learn about the Storefront Compatibility Package, which provides GraphQL enhancements available to your Storefront.
+sidebar:
+  order: 7
+---
+
+import Tasks from '@components/Tasks.astro';
+import Task from '@components/Task.astro';
+import Callouts from '@components/Callouts.astro';
+import Aside from '@components/Aside.astro';
+import { Tabs, TabItem, Steps } from '@astrojs/starlight/components';
+
+
+The Storefront Compatibility Package contains changes to the Adobe Commerce 2.4.7 codebase that enable dropin functionality. Most of these changes enhance the Adobe Commerce GraphQL schema. However, miscellaneous bugfixes are also included.
```

Comment:
call out that core commerce dropins that power low funnel experience (Cart/checkout/Accoutn/ Auth) require SCP